INSTALL.md

```diff
@@ -84,7 +86,7 @@ bin/auto_run_tests.pl
 **For Windows:**
 
 ```
-bin\auto_run_tests.pl
+perl bin\auto_run_tests.pl
```

Comment:
Not sure why this is added?

Reply:
I think I added this because this didn't work on Windows when I tried it. Is it supposed work without using `perl`?

Reply:
Yes, assuming Perl is correctly installed.  I've used this with both ActiveState and Strawberry.